amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -2091,7 +2094,163 @@ paths:
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission to create event
+        '409':
+          description: Cannot create event. An event with the name **name** already exists
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  
+  '/events/{eventId}':
+    get:
+      tags:
+        - events
+      summary: Return event for given eventId
+      description: |
+        Get information about the retention event with id **eventId**.
+      operationId: getEvent
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/eventIdParam'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/EventEntry'
+        '401':
+          description: Authentication failed
+        '404':
+          description: "**eventId** does not exist"
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    put:
+      tags:
+        - events
+      summary: Update event for given eventId
+      operationId: updateEvent
+      description: |
+        Update retention event with id **eventId**.
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/eventIdParam'
+        - in: body
+          name: eventBodyUpdate
+          description: The event information to update.
+          required: true
+          schema:
+            $ref: '#/definitions/EventBodyUpdate'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/EventEntry'
+        '400':
+          description: |
+            Invalid parameter: The update request is invalid or **eventId** is not a valid format or **eventBodyUpdate** is invalid
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission to update events
+        '404':
+          description: "**eventId** does not exist"
+        '409':
+          description: Cannot update event. An event with the name **name** already exists
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  
+  '/event-types':
+    get:
+      tags:
+        - events
+      summary: List all the retention event types
+      description: |
+        Get a list of all the retention event types.
+      operationId: getAllEventTypes
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/skipCountParam'
+        - $ref: '#/parameters/maxItemsParam'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/EventTypePaging'
```

Comment:
`400` error code is missing. Similar to my earlier comment: 

> Since you are supporting paging, ...

Reply:
Done. Raised below Jira ticket: https://alfresco.atlassian.net/browse/RM-7186